Dockerfile.windows

```diff
@@ -208,6 +211,40 @@ RUN `
     } `
   } `
   `
+  Function Build-GoTestSum() { `
+    $GotestsumPath=Join-Path -Path $Env:GOPATH -ChildPath "src\gotest.tools\gotestsum"; `
+    $GotestsumBinPath=Join-Path -Path $GotestsumPath -ChildPath "gotestsum.exe"; `
+    Write-Host "INFO: Building gotestsum version $Env:GOTESTSUM_COMMIT in $Env:GOPATH" `
+    $optsForGet = @('"get"', '"-d"', '"gotest.tools/gotestsum"'); ` 
+    $logFileName = 'gotestsum.get.out';  `
+    &go $optsForGet; `
+    $savedExitCode = $LASTEXITCODE; `
+    if ($savedExitCode -ne 0) {  `
+      Throw '"Failed getting gotestsum sources..."'  `
+    }; `
+    Write-Host "INFO:     Sources obtained for gotestsum..."; `
+    Push-Location $GotestsumPath; `
+    $optsForCheckout = @('"checkout"', '"-q"', """$Env:GOTESTSUM_COMMIT"""); ` 
+    $logFileName = 'gotestsum.git.out';  `
```

Comment:
This variable is no longer used so can be removed

Reply:
Done